types/weixin-app/index.d.ts

```diff
@@ -0,0 +1,2155 @@
+// Type definitions for wx-app 1.0
+// Project: https://mp.weixin.qq.com/debug/wxadoc/dev/api/
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace wx {
+	type NoneParamCallback = () => void;
+	type ResponseCallback = (res: any) => void;
```

Comment:
Again, the type aliases here aren't really helping; just do it inline as in `interface BaseOptions { success?(successValueNameHere: SuccessTypeNameHere): void; ... }`.